src/pages/docs/Connecting to Databases/ui.mdx

```diff
@@ -0,0 +1,75 @@
+---
+name: Using Database Connection UI
+menu: Connecting to Databases
+route: /docs/databases/db-connection-ui
+index: 1
+version: 1
+---
+
+**Add Giph for before and after**
+
+There are now 3 steps when connecting to a database:
```

Comment:
another nit.. I'd recommend: `There are now 3 steps when connecting to a database in the new UI.`